CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# Build
+The declaration files are auto generated(as there are new corvid APIs on a monthly base) 
```

Comment:
Grammer - Monthly basis.
Perhaps not even monthly though, I would say (as new apis are added all the time, or something like that. Feel free to consult sam about the english)